tests/integration/transaction_record_query_e2e_test.py

```
@@ -142,4 +142,31 @@ def test_transaction_record_query_can_execute_fungible_transfer():
         assert record.transaction_memo == "", "Transaction memo should be empty by default"
         assert record.transaction_hash is not None, "Transaction hash should not be None"
     finally:
-        env.close()
\ No newline at end of file
+        env.close()
+@pytest.mark.integration
+def test_query_with_include_duplicates():
+    client = Client.for_testnet()  # or use IntegrationTestEnv().client if project has it
```

Ruff (0.14.14)

[error] Undefined name `Client`

(F821)